src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -145,7 +145,10 @@ def get(self, request: Request, organization) -> Response:
                     queryset = queryset.filter(role__in=[r.id for r in roles.with_any_scope(value)])
 
                 elif key == "role":
-                    queryset = queryset.filter(role__in=value)
+                    members_with_role = organization.get_members_with_org_roles(
+                        roles=value, include_null_users=True
+                    ).values_list("id", flat=True)
```

Comment:
Is the idea behind `include_null_users=True` that we want the API to include unanswered invites? If so, what does the "role" key mean in that context? Would those always be the default (lowest) role?

Reply:
you can specify what role you want a member to have when you invite them, and sometimes we want to include these members when querying to members with a specific role

Reply:
so yes, unanswered invites, but unanswered invites still have a specific role set

Reply:
Makes sense, thanks.